Task 1/index.js

```diff
@@ -0,0 +1,38 @@
+/* 
+Дан масив об'єктів. Вивести масив телефонних номерів користувачів, у яких баланс більше 2000 доларів. 
+І знайти суму всіх балансів користувачів
+*/
+
+import users from './data/data.json' assert { type: 'json' };
+import { convertDollarsToNumber } from './helpers/helper.js';
+
+const filterUserPhonesByBalance = (value, index) => {
+	const generateFilterArr = [];
+
+	if (Number.isInteger(index) && Array.isArray(value)) {
+		for (let i = 0; i < value.length; i++) {
```

Comment:
Not bad. But can you rewrite this loop with chaining array methods `Array.filter().map()`